examples/Generic/src/main/java/org/astarteplatform/devicesdk/generic/examples/ExampleDevice.java

```diff
@@ -59,9 +64,23 @@ public static void main(String[] args) throws Exception {
     }
 
     String realm = cmd.getOptionValue("realm");
+    String pairingUrl = cmd.getOptionValue("pairing-url");
     String deviceId = cmd.getOptionValue("device-id");
     String credentialsSecret = cmd.getOptionValue("credentials-secret");
-    String pairingUrl = cmd.getOptionValue("pairing-url");
+    String jwtToken = cmd.getOptionValue("jwt-token");
+
+    if (deviceId == null || deviceId.isEmpty()) {
+      /*
+       * Astarte device id creation
+       */
+
+      UUID uuidNameSpace = UUID.fromString("f79ad91f-c638-4889-ae74-9d001a3b4cf8");
+      deviceId = AstarteDeviceIdUtils.generateId(uuidNameSpace, "001122334455");
```

Comment:
again, maybe using an example mac address might be more informative for our users.
So they can guess when UUIDv5 is better for them.